q2_feature_table/_core_features/core_features_assets/index.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<h1>Core features</h1>
+
+<div class='row'>
+  <div class='col-md-6'>
+    <div class='row'>
+      <div class='col-md-12'>
+        <h3>Feature table summary</h3>
+        Sample count: {{num_samples}}<br>
+        Feature count: {{num_features}}
+      </div>
+    </div>
+    <div class='row'>
+      <div class='col-md-12'>
+        <img src='core-feature-counts.svg'>
+      </div>
+    </div>
+  </div>
+  <div class='col-md-6'>
+    <div class='row'>
+      <div class='col-md-12'>
+        <h3>Terminology key</h3>
+        <i>Fraction of samples</i>: The fraction of the total number of samples that a feature must be observed more than zero times in for that feature to be considered "core".<br>
+        <i>Feature count</i>: The number of "core" features at a given fraction of samples.<br>
+        <i>Fraction of features</i>: The fraction of the total number of features that are "core" features at a given fraction of samples.<br>
+        <i>Feature list</i>: The list of "core" feature ids and their percentile frequencies at a given fraction of samples.
```

Comment:
I think the current wording makes sense -- I would also be open to changing it to `frequencies for a given fraction of samples`, both wordings seem equally clear to me. Using `across` doesn't make sense because each term (i.e. column label) is referring to core features at a single fraction of samples -- IOW, each row represents core features at a single fraction of samples (i.e. a single *step*). `across` seems to imply that the core features span fractions of samples. Does that make sense?